data/examples/rasa/demo-rasa-multi-intent.md

```diff
@@ -0,0 +1,71 @@
+## intent:affirm
```

Comment:
in the test `"intent_split_symbol": "+"` but there is no such intent in this file

Reply:
äh... I changed the chitchat intents, strange, will do the changes again

Reply:
`/` in chitchat intents is a special character to indicate response selection

Reply:
yeah, but we do not test response selection in this test and the data file is only used in this one test, so should be fine to replace it